asllib/Interpreter.ml

```diff
@@ -1009,16 +1017,21 @@ module Make (B : Backend.S) (C : Config) = struct
         in
         Error.fatal_unknown_pos (Error.UncaughtException msg)
 
+  let run_typed ast env = run_typed_env [] ast env
+
   (** Main entry point for the Interpreter, [run ast primitives] type-annotate
       [ast], build a global environment and then evaluate the "main" function
       in it. *)
```

Comment:
I think we can delete those at this point

Reply:
Delete comment?

Reply:
Yes: they are not alongside their function as you put run_env between them, and do not have the correct arguments anymore.